data-structures/Trivial Hashing

```diff
@@ -0,0 +1,52 @@
+// CPP program to implement direct index mapping 
+// with negative values allowed. 
+#include <bits/stdc++.h> 
+using namespace std; 
+#define MAX 1000 
+
+// Since array is global, it is initialized as 0. 
+bool has[MAX + 1][2]; 
+
+// searching if X is Present in the given array 
+// or not. 
+bool search(int X) 
+{ 
+	if (X >= 0) { 
+		if (has[X][0] == 1) 
+			return true; 
+		else
+			return false; 
+	} 
+
+	// if X is negative take the absolute 
+	// value of X. 
+	X = abs(X); 
+	if (has[X][1] == 1) 
+		return true; 
+
+	return false; 
+} 
+
```

Comment:
Please use this code sytle conventions
https://google.github.io/styleguide/cppguide.html
https://google.github.io/styleguide/cppguide.html#Formatting